src/backends/dlib/dliblib.cc

```diff
@@ -37,6 +37,7 @@ namespace dd {
             :MLLib<TInputConnectorStrategy, TOutputConnectorStrategy, DlibModel>(std::move(cl)) {
         this->_libname = "dlib";
         _net_type = cl._net_type;
+        this->_mltype = "detection/classification (" + _net_type + ")";
```

Comment:
I recommend we keep the `mltype` identical to the existing ones, e.g. from the Caffe backend.

Reply:
👍 I'll change this to simply `detection`. I noticed afterwards that a `GET /services/<serviceName>/info` returns service-specific parameters anyways, which will show the `_net_type` there. This `_mltype` I believe is shown for `GET /services/info`.